src/algebra/big_operators/norm_num.lean

```diff
@@ -166,13 +171,21 @@ meta def eval_multiset : expr → tactic (list expr × expr)
   eis ← (list.range n).mmap (λ i, expr.of_nat `(ℕ) i),
   eq ← mk_eq_refl e,
   pure (eis, eq)
+| `(@@coe (@@coe_to_lift (@@coe_base (multiset.has_coe))) %%exs) := do
+  (xs, xs_eq) ← eval_list exs,
+  eq ← i_to_expr ``(congr_arg coe %%xs_eq),
+  pure (xs, eq)
```

Comment:
Is it much work to add a norm_num test for these new cases?

Reply:
It's implicitly covered by the existing tests in `test/norm_num_ext.lean`. I can try and figure out an explicit test if you want.

Reply:
I think it would be good to add an explicit test for each of these new cases to

https://github.com/leanprover-community/mathlib/blob/kill_fin_range/test/norm_num_ext.lean#L271-L290